packages/patternfly-4/react-charts/src/components/ChartDonut/examples/ChartDonut.md

```diff
@@ -123,15 +125,15 @@ import { ChartDonut } from '@patternfly/react-charts';
       width={300}
     />
   </div>
-</div>
+)
 ```
 
-```js title=Small-donut-chart
+```js title=Small
```

Comment:
I feel we need to leave "donut chart" in the title as "Small" by itself is not descriptive of what is small; the donut chart, legend, text, etc.? The title was "Small donut chart"

Reply:
See conversation on Basic/Simple titles.